content/events/2024-in-person-conference/_index.md

```diff
@@ -5,22 +5,43 @@ description = "Nordic RSE – Research Software Engineers in the Nordics – Con
 
 # Nordic-RSE in person conference 2024 <span style="color: gray;">#NordicRSEconf</span>
 
+
 **May 30-31**
 
 Registration will open in January 2024
 
 
-## Call for Contributions
+## What is it?
+
+This is the first Nordic RSE in person conference. We are an association of researchers who use software and people who write
+that software. So this is not like most academic conferences in that the focus is not on the results of research, but rather
+on the tools that underlie those results.
+
+Most research is already built on a foundation of sofware tools and the quality and reliability of those tools is key to
+producing reliable results. And the importance of software keeps growing. In this conference we want to talk about what software
+you are use for your research and how you use it, the software you develop and how you go about developing it.
+
 
-We will soon open a call for contributions.
- 
+## Where
 
-You can also help us by [sharing the announcement with others](/events/2023-online-unconference/share/). 
+At Aalto University, Espoo, Finland.
 
-And of course we will all follow a [Code of
-Conduct](https://nordic-rse.org/about/code-of-conduct/).
 
-All formats of communication are welcome (or check what we have done [last year](/events/2023-online-unconference/)):
+### How you can help
+
+ - Submit a contribution! The conference will be more interesting with your contribution on the agenda.
+ - [Share the announcement with others](/events/2023-online-unconference/share/). 
```

Comment:
```suggestion
 - Submit a contribution! The conference will be more interesting with your contribution on the agenda.
 - Encourage your colleagues to submit a contribution. Since the term RSE is still fairly new, they might not know about it yet and be hesitant to submit something. 
 - [Share the announcement with others](/events/2023-online-unconference/share/). 
```